server/db/ride.sql

```diff
@@ -1,14 +1,14 @@
 CREATE TABLE rides (
     id SERIAL PRIMARY KEY,
-    destinationStartPoint VARCHAR NOT NULL,
-    destinationStopPoint VARCHAR NOT NULL,
+    startPoint VARCHAR NOT NULL,
+    stopPoint VARCHAR NOT NULL,
     departureTime VARCHAR NOT NULL,
```

Comment:
You should also have the departureDate too, I remember they gave you the feedback.

Reply:
fixed